configTools/src/test/scala/com/gu/janus/policy/IamSpec.scala

```diff
@@ -0,0 +1,269 @@
+package com.gu.janus.policy
+
+import com.gu.janus.policy.Iam._
+import io.circe.JsonObject
+import io.circe.parser.decode
+import io.circe.syntax._
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+
+class IamSpec extends AnyFreeSpec with Matchers {
```

Comment:
All the other tests use the `XXTest` naming convention (rather than `XXSpec`). Minor point, but it might make it easier to jump to files if we're consistent with how they're named?

Reply:
Renamed in b736c07f2cc05de8c671688b276e02f90f5e7b72